src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write your code here
+
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+const population = document.getElementsByClassName('population');
+
+const data = [];
+
+for (const el of population) {
+  const tmp = el.innerText;
+
+  data.push(parseInt(tmp.replaceAll(',', '')));
```

Comment:
It's a good idea to check if `parseInt` has successfully parsed the number. If `parseInt` fails, it will return `NaN`, which may lead to incorrect calculations in the `reduce` function.